.github/bors.toml

```diff
@@ -0,0 +1,7 @@
+status = [
+    "ci",
```

Comment:
I've seen that bors only posts the link to the `ci` job. I was wondering if it's possible to make it post the link to all the involved runs? I'll make the interesting content a couple clicks closer, and, afaik, it'll also embed the success state of each of the individual job - thus providing us a summary of the test results.

Might this setting affect that?

Reply:
No - this setting determines whether the CI passed or failed and whether it can be merged.

Reply:
I wonder why it's an array...

Can't we just put the list of all the jobs there:
```toml
"test-linux",
"build-x86_64-unknown-linux-gnu",
"build-aarch64-unknown-linux-gnu",
"test-mac",
"test-windows",
"test-misc",
"test-integration-aws",
"test-integration-clickhouse",
"test-integration-docker",
"test-integration-elasticsearch",
"test-integration-gcp",
"test-integration-humio",
"test-integration-influxdb",
"test-integration-kafka",
"test-integration-loki",
"test-integration-pulsar",
"test-integration-splunk",
"build-deb-package",
"compute-k8s-test-plan",
"test-e2e-kubernetes",
"check-component-features",
```